src/foam/nanos/auth/ViewPropertyConfig.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license
+ * Copyright 2023 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.CLASS({
+  package: 'foam.nanos.auth',
+  name: 'ViewPropertyConfig',
```

Comment:
Does this even need to be modelled? We typically just use a Map of Maps for this behaviour as seen in [AstractSectionedDetailView ](https://github.com/kgrgreer/foam3/blob/5ea78878ebff1f195210448bbe88f25638b21335/src/foam/u2/detail/VerticalDetailView.js#L18-L22) So in this case this model can be discarded and the AddressConfig model can have only two properties for countryId and config Map property

Reply:
This was intentional, to capture additional 'view' behaviour going forward.